### ISSUE TITLE
"Documenting Coq with Sphinx" does not say how to document library paths

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#10761]
#### Description of the problem

https://github.com/coq/coq/blob/master/doc/sphinx/README.rst does not seem to say how to quote things like `Coq.Init.Tactics` and `Coq.Program.Tactics` in rst files

#### Coq Version

cc93f41

**Comment by JasonGross** [comment]
cc @coq/doc-maintainers ?

**Comment by cpitclaudel** [comment]
Good point.  We don't have anything better than `` :g:`…` `` at the moment.  It would be nice to have a role that would hyperlink to the library documentation.

**Comment by Zimmi48** [comment]
See also #10142.
